libraries/cms/html/contentlanguage.php

```diff
@@ -54,9 +81,42 @@ public static function existing($all = false, $translate = false)
 			static::$items = $db->loadObjectList();
 		}
 
+		return static::$items;
+	}
+
+	/**
+	 * Get a list of the available allowed content language items.
+	 *
+	 * @param   boolean  $all        True to include All (*)
+	 * @param   boolean  $translate  True to translate All
+	 *
+	 * @return  string
+	 *
+	 * @see     JFormFieldContentLanguage
+	 * @since   1.6
```

Comment:
Well, as we will not change that field type for the moment, we will create a new field type to be used by com_associations